src/OpenTelemetry/Metrics/PullMetricScope.cs

```diff
@@ -7,7 +7,7 @@ namespace OpenTelemetry.Metrics;
 
 internal sealed class PullMetricScope : IDisposable
 {
-    private static readonly RuntimeContextSlot<bool> Slot = RuntimeContext.RegisterSlot<bool>("otel.pull_metric");
+    private static readonly RuntimeContextSlot<bool> Slot = RuntimeContext.RegisterSlot<bool>("otel.pull_metric")!;
```

Comment:
I think this one needs a closer look.
I'm not familiar with the RuntimeContext api. If the RegisterSlot method can return a nullable, that might suggest something failed.

Reply:
Fixed in fe4d9bf9e1740bdd673aa0ae68281d1678f39bdb

I double checked. There is no possibility to return null (at least as long someone does no play with reflection with `contextSlotType` field in `RuntimeContext` class.